src/vaults/factories/TokenFactory.sol

```diff
@@ -46,7 +46,11 @@ contract TokenFactory is Auth, ITokenFactory {
                 bytes1(0xff),
                 address(this),
                 salt,
-                keccak256(abi.encodePacked(type(CentrifugeToken).creationCode, abi.encode(decimals)))
+                keccak256(
+                    abi.encodePacked(
+                        type(CentrifugeToken).creationCode, abi.encode(decimals), abi.encode(address(this))
+                    )
+                )
```

Comment:
I think this change serves as argument to revert the changes for factory-deployed contracts. We must not change `getAddress` for factories which we already use in practice because this breaking change is not backwards compatible with our legacy Maker and Morpho vaults, i.e. `getAddress` now would return different a different deterministic address.

Reply:
I think you're right. I'm gonna revert the changes that has to do with factories. We only need to support Create3 in contracts directly deployed by our scripts

Reply:
Also it reduces a bit of noise

Reply:
Yes, factories can still deploy with `Create2` confirmed by Jeroen ([ref](https://kflabs.slack.com/archives/C07PG2EUR9C/p1744709568426489?thread_ts=1744709101.655979&cid=C07PG2EUR9C)).